.gitignore

```diff
@@ -122,7 +122,7 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
+#lib/
```

Comment:
nit: remove comment

(this is a joke don't kill me lmao)

Reply:
The reason why I kept it is because if this happens with a language in the future it can be enabled again. We can remove it though.